things/service/src/main/java/org/eclipse/ditto/things/service/utils/PointerUtils.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2025 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.ditto.things.service.utils;
+
+import org.eclipse.ditto.json.JsonPointer;
+
+/**
+ * Utility class for JsonPointer operations.
+ * Provides consistent handling of pointer paths without raw string manipulation.
+ */
+public final class PointerUtils {
+
+    private static final JsonPointer ROOT_POINTER = JsonPointer.of("/");
+
+    private PointerUtils() {
+        // No instantiation
+    }
+
+    /**
+     * Checks if a pointer represents the root path.
+     *
+     * @param pointer the pointer to check
+     * @return true if the pointer is root (empty or "/")
+     */
+    public static boolean isRoot(final JsonPointer pointer) {
```

Comment:
Method is not used, can be removed.

Also, `pointer.isEmpty()` is equivalent to `pointer.equals(ROOT_POINTER)` - does not make much sense.